### PR TITLE
Update the integration Dockerfile to reduce the image size

### PR DESCRIPTION
**Pull request by firestarman**
-   Assembled all the installations in one RUN
-   Cleaned the conda cache files after all the installations done
-   Removed the duplicate python3.6

Signed-off-by: Firestarman <firestarmanllc@gmail.com>

### CODE REVIEW

**Comment by firestarman** [comment]
Verified locally and all the IT job passed

**Comment by pxLi** [comment]
build

**Review by pxLi — APPROVED** [review]
LGTM. But require approval from [CODEOWNERS](https://github.com/NVIDIA/spark-rapids/blob/branch-0.3/.github/CODEOWNERS)

**Review by NvTimLiu — APPROVED** [review]

**Review by revans2 — APPROVED** [review]

**Pull request merged**
